tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -210,7 +213,75 @@ void test_ricci_scalar(const Solution& solution,
       inverse_spatial_metric);
 
   CHECK_ITERABLE_APPROX(get(ricci_scalar), expected(get(ricci_scalar).size()));
-}  // namespace
+}
+
+template <typename Solution, typename ExpectedLambda>
+void test_area_element(const Solution& solution, const double& surface_radius,
+                       const ExpectedLambda& expected) noexcept {
+  const auto box =
+      db::create<db::AddTags<StrahlkorperTags::items_tags<Frame::Inertial>>,
+                 db::AddComputeItemsTags<
```

Comment:
When you rebase on develop note that: AddTags -> AddSimpleTags and AddComputeItemsTags -> AddComputeTags

Reply:
OK, I took out the ampersand, rebased against develop, made this fix, and squashed